packages/ui-components/src/components/Matrix/Matrix.tsx

```diff
@@ -16,6 +16,7 @@ import { MatrixLegend } from './MatrixLegend';
 const MatrixTable = styled.table`
   color: ${({ theme }) => theme.palette.text.primary};
   height: 1px; // this is to make the cell content (eg. buttons) take full height of the cell, and does not actually get applied
+  border-collapse: collapse; // necessary to enable borders between column groups
```

Comment:
Hm, would it be disruptive to do this at the `CssBaseline` level? I can’t imagine any scenario where we wouldn’t want to set `collapse`, but you’ve got a lot more familiarity with our many tables 😅

Reply:
I think in most cases, when we have sticky headers we want it to be `separate` because this makes the borders stay visible when scrolling. But in this case, it had to be set as `collapse` because `colgroup` borders are only visible in when `border-collapse` is `collapse`. Annoyingly they don't both work on the same setting 😓

Reply:
Now that you say that, I think I have asked this question before. Sorry and thank you!